### PR TITLE
chore: updated dependencies (#3372)

### LINKED ISSUE #3372: Update frontend dependencies
The frontend dependencies haven't been updated in a long time, and there are pending vulnerability reports.

The dependencies should be updated, and there should be no [pending reports](https://github.com/Kotlin/dokka/security/dependabot).

### PR DESCRIPTION
**Pull request by berezinant**
fixes #3372

### CODE REVIEW

**Review by IgnatBeresnev — APPROVED** [review]
YOLO

(let's wait until code freeze to merge this)

**Review by zoobestik — APPROVED** [review]

**Pull request merged**
